lib/thumbs/pull_request_worker.rb

```diff
@@ -702,7 +702,10 @@ def create_build_status_comment
 
 <%= status[:command] %>
 
-<%= status[:output] %>
+<% output=status[:output] %>
+<% allowed_length=10000 %>
+<%= output.slice!(output.length-allowed_length, output.length) %>
```

Comment:
regarding the slice, the intent is that it slices the preceding characters of the amount that is over the limit, so it always shows the tail.